README.md

```diff
@@ -46,6 +46,29 @@ while running:
 c.close()
 ```
 
+**AvroProducer**
+```
+from confluent_kafka import Producer
+from confluent_kafka.avro.AvroProducer import AvroProducer
+from confluent_kafka.avro.CachedSchemaRegistryClient import CachedSchemaRegistryClient
+from confluent_kafka.avro.serializer import Util
+from confluent_kafka.avro.serializer.MessageSerializer import MessageSerializer
+
+value_avro_schema = Util.parse_schema_from_file('ValueSchema.avsc')
+key_avro_schema = Util.parse_schema_from_file('KeySchema.avsc')
+value = {"name": "Value"}
+key = {"name": "Key"}
+
+producer = Producer({'bootstrap.servers': 'mybroker,mybroker2'})
+
+schemaRegistryUrl = 'https://<host>:<port>'
+schemaRegistryClient = CachedSchemaRegistryClient(url=schemaRegistryUrl)
+serializer = MessageSerializer(schemaRegistryClient)
+
+avroProducer = AvroProducer(producer, serializer)
```

Comment:
@ewencp I started with inheritance, but it did not let me inherit class Producer. 
```
Traceback (most recent call last):
  File "test.py", line 4, in <module>
    class AvroProducer(Producer):
TypeError: Error when calling the metaclass bases
    type 'cimpl.Producer' is not an acceptable base type
```